apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Snippets/SubscriberClientSnippets.cs

```diff
@@ -36,5 +41,92 @@ public void Emulator()
             // End sample
         }
 
+        [Fact]
+        public void AddSubscriberClient()
+        {
+            string projectId = "projectId";
+            string subscriptionId = "subscriptionId";
+            var services = new ServiceCollection();
+
+            // Sample: AddSubscriberClient
+            SubscriptionName subscriptionName = SubscriptionName.FromProjectSubscription(projectId, subscriptionId);
+            services.AddSubscriberClient(subscriptionName);
+            // End sample        
+        }
+
+        [Fact]
+        public void AddCustomizedSubscriberClient()
+        {
+            string projectId = "projectId";
+            string subscriptionId = "subscriptionId";
+            var services = new ServiceCollection();
+
+            // Sample: AddCustomizedSubscriberClient
+            SubscriptionName subscriptionName = SubscriptionName.FromProjectSubscription(projectId, subscriptionId);
+            services.AddSubscriberClient(builder =>
+            {
+                builder.SubscriptionName = subscriptionName;
+                builder.CredentialsPath = "path/to/credentials.json";
+                // Other settings to customize the client.
+            });
+            // End sample
+        }
+
+        [Fact]
+        public void AddHostedService()
+        {
+            var services = new ServiceCollection();
+
+            // Sample: AddHostedService
+            services.AddHostedService<SubscriberService>();
+            // End sample        
+        }
+
+        [Fact]
+        public async Task UseSubscriberServiceInConsoleApp()
+        {            
+            string projectId = "projectId";
+            string subscriptionId = "subscriptionId";
+
+            // Sample: UseSubscriberServiceInConsoleApp
+            // Add `using Microsoft.Extensions.Hosting;` in the using directives.
+            // Below code goes inside the Main method, i.e., public static async Task Main(string[] args)
```

Comment:
I'd remove this line to be honest - especially as with minimal API templates, people may well be using top-level statements anyway, where the Main method is implicit.

(It's not big deal if you decide to leave it though. I'd reword "Below code" to "The code below" though.)